### PR TITLE
Add unit tests and bug fixes in existing files

### PR DESCRIPTION
**Pull request by VolodymyrFihurniak**

### CODE REVIEW

**Review by denbykov — COMMENTED** [review]
Fix found issues

File: server/src/data/FileRepository.ts [hunk 1]
```diff
@@ -144,11 +144,11 @@ export class FileRepository implements iFileDatabase {
       dataLogger.debug(queryUserFiles);
       await client.query(queryUserFiles, [userId, fileId]);
       new PublisherAMQP(config).publishDownloadingQueue(queueDownloading, {
-        type: queueDownloading,
+        type: 'get_file/youtube',
```
Comment (denbykov) [inline server/src/data/FileRepository.ts:147]: You need to use configuration variables here

File: server/src/data/FileRepository.ts [hunk 2]
```diff
@@ -144,11 +144,11 @@ export class FileRepository implements iFileDatabase {
...
         fileId: fileId,
       });
       new PublisherAMQP(config).publishDownloadingQueue(queueParsing, {
-        type: queueParsing,
+        type: 'set_tags/youtube-native',
```
Comment (denbykov) [inline server/src/data/FileRepository.ts:151]: You need to use configuration variables here

File: server/src/entities/tag.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export class Tag {
   public year: Date;
   public trackNumber: number;
   public sourceType: { id: number; description: string };
-  public statusId: { id: number; description: string };
+  public status: { id: number; description: string };
```
Comment (denbykov) [inline server/src/entities/tag.ts:11]: Does this field have an id?

File: server/src/entities/tag.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ export class Tag {
     year: Date,
     trackNumber: number,
     sourceType: { id: number; description: string },
-    statusId: { id: number; description: string }
+    status: { id: number; description: string }
```
Comment (denbykov) [inline server/src/entities/tag.ts:22]: Does this field have an id?

**Review by denbykov — DISMISSED** [review]

**Review by denbykov — COMMENTED** [review]
image
Routes should be specified inside routes classes, not controller

**Comment by VolodymyrFihurniak** [comment]
> image Routes should be specified inside route classes, not controller.

It has been resolved. Moved routes to route configuration files

**Review by denbykov — APPROVED** [review]

**Pull request merged**